package.json

```
@@ -10,6 +10,7 @@
     "deploy/",
     "export/",
     "scripts/",
+    ".openzeppelin/mainnet.json",
```

Can we define it as a directory, to not be tied to the mainnet?

```suggestion
    ".openzeppelin/",
```

---

OpenZeppelin produces upgradeability manifest files for each network (see https://docs.openzeppelin.com/upgrades-plugins/1.x/network-files#network.json). For example, I have locally a `.openzeppelin/unknown-31337.json` file, generated when I run the upgrades, and I don't want this file to be packaged. So, for the moment, I think it's better to tie it just to mainnet, as we only care for upgradeability there.